Loop Status Extension/StatusViewController.swift

```diff
@@ -205,11 +205,24 @@ class StatusViewController: UIViewController, NCWidgetProviding {
                 if let eventualGlucose = predictedGlucose.last {
                     let formatter = NumberFormatter.glucoseFormatter(for: eventualGlucose.unit)
 
+                    var activeInsulinString = "--"
+                    if let activeInsulin = context.activeInsulin {
+                        // If the returned value doesn't indicate an error,
+                        // assign it to the label:
+                        if activeInsulin.error == "" {
+                            activeInsulinString = String(format: "%.1f", activeInsulin.value)
+                        } else {
+                            // This will be ugly but maybe (?) informative: 
+                            activeInsulinString = activeInsulin.error
+                        }
+                    }
+
                     if let eventualGlucoseNumberString = formatter.string(from: NSNumber(value: eventualGlucose.quantity.doubleValue(for: unit))) {
                         subtitleLabel.text = String(
                             format: NSLocalizedString(
-                                "Eventually %1$@ %2$@",
-                                comment: "The subtitle format describing eventual glucose. (1: localized glucose value description) (2: localized glucose units description)"),
+                                "%1$@ U IOB;   Eventual BG %2$@ %3$@",
```

Comment:
The preferred wording here is "Eventually", to match the app.

Reply:
That's what I had originally, but here I think the situation is a bit different because of the addition of the IOB value.  In the app, it's clear that the number after "Eventually" refers to BG, since there it says "Glucose" earlier on the line and it's right above the glucose graph (so the full text is "Glucose   Eventually 100 mg/dl".  The placement above the glucose graph is the same here, but it's complicated by having the IOB there, too.  So if we follow the app, you'd get:

 0.6 U IOB;  Eventually 100 mg/dl

which to me doesn't make as much sense at a glance as 

 0.6 U IOB;   Eventual BG 100 mg/dl

since we've reported IOB early in the string and then we're saying "eventually" another number.  No one would *really* confuse that for being the eventual IOB, I don't think, but it does seem incongruous to me.  

What do you think?   And while we're on formatting, I'll point out another small thing: originally I had the number and the "IOB" switched, like this:

 IOB 0.6 U;   Eventual BG 100 mg/dl

but I found that in my actual use, the glanceability was slightly worse; I spent just an extra fraction of a second mentally finding that value in the middle of the string.  With it all the way to the left, it's definitely easier to mentally process at a glance.  

Similarly, I initially had two decimal places, matching the app, but decided that again, the mental information processing was simpler with a cleaner number there, and that the IOB in the hundredths place wasn't meaningful (i.e. the rounding of 0.05 U of insulin won't matter for me, though I suppose it could for someone who is super sensitive).   That one I don't feel as strongly about; could go back to two places here as a default if you think that's better. 

Just FYI about some small design decisions in case you're wondering about that as we discuss format here.   Both of these were usability / legibility related for me.